packages/cli/README.md

```diff
@@ -34,6 +34,17 @@ glaze COMMAND
 - [`glaze model:inspect NAME`](#glaze-modelinspect-name)
 - [`glaze model:list`](#glaze-modellist)
 - [`glaze model:publish NAME [OUTPUT]`](#glaze-modelpublish-name-output)
+- [`glaze stream:pin:add STREAMID`](#glaze-streampinadd-streamid)
+- [`glaze stream:pin:rm STREAMID`](#glaze-streampinrm-streamid)
+- [`glaze stream:pin:ls STREAMID`](#glaze-streampinls-streamid)
```

Comment:
Could we make these `pin:*` rather than `stream:pin:*`?

Reply:
100%. Same for Schema (pending removal)